CHANGES.txt

```diff
@@ -35,6 +35,10 @@ Changes
 Fixes
 =====
 
+ - Fixed validation so that ``SELECT DISTINCT`` can be used only if there
+   is no ``GROUP BY`` present or if all the symbols of select list are exactly
+   the same as the ones in group by, in any order.
```

Comment:
```
Fixed validation so that ``SELECT DISTINCT`` can be used only if there
is no ``GROUP BY`` present or if the set of ``GROUP BY`` terms is the same as
the set ``SELECT`` terms.
```

using sets here, which implies ordering doesn't matter (i.e. not lists)

is terms correct? or is it defs "symbols"